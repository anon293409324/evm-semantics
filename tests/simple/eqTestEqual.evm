account:
-   id: 1
-   balance: 40
-   program: PUSH 3 ; PUSH 3 ; EQ
-   storage:

.Accounts


transaction:
-   to: 1
-   from: .AcctID
-   data:
-   value: 0
-   gasPrice: 1
-   gasLimit: 100

.Transactions
