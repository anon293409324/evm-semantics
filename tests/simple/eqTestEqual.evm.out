<T> <k> . </k> <id> 1 </id> <pc> 3 </pc> <gas> 91 </gas> <wordStack> 1 : .WordStack </wordStack> <localMem> .Map </localMem> <gasPrice> 1 </gasPrice> <callStack> .CallStack </callStack> <accounts> <account> <acctID> 1 </acctID> <program> 0 |-> ( PUSH 3 ) 1 |-> ( PUSH 3 ) 2 |-> EQ </program> <storage> .Map </storage> <balance> 40 </balance> </account> </accounts> </T>
